### DISCUSSION TITLE
Pushes do not Show on my contribution board

### DISCUSSION THREAD

**Discussion by AleksBulajic** [community/community#68520] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello everyone!!
I'v been having issues with the contribution board, every time  commit new code to my repositories it doesn't not appear to my contribution board ( I do not see new green dots added )

Does anyone know hot to fix this?

**Comment by vishalkrishna8** [comment, marked as answer]
Hello @AleksBulajic ,
If your GitHub contributions aren't appearing on your contribution graph, Here are some steps to help troubleshoot and potentially resolve this issue:

1.Check your commit email and ensure it matches your GitHub email.
2.Contributions to private repositories won't appear on the public graph.
3.Commits to forks may not count unless merged into the original repo.
4.Allow time for updates, as the graph might have a delay.
5.Confirm repository settings and services aren't disabled.
6.Check your contribution calendar on your profile.
7.Ensure your Git config has the correct name and email.
8.Avoid force pushes, as they can remove contributions.
9.Verify there are no network issues.
10.Check GitHub's status page for any ongoing issues.


Let me know if this was helpful!

**Reply by AleksBulajic** [reply]
Hello @vishalkrishna8 

thank you for responding and giving be this steps to follow to troubleshoot my issues, but unfortunately everything on the list seems correct on my end, 
The only commits that i see appear to my contribution board are when i edit the readme.md. I wonder what could be the problem.

**Reply by AleksBulajic** [reply]
Noticed that only the commits that are showing are the ones when i edit code thru the github, but everything i commit and push thru my local files dont appear as a green dot

**Reply by vishalkrishna8** [reply]
Contributions are linked to the email address associated with your GitHub account. Ensure that the email address used for your Git commits matches the email associated with your GitHub account. 

(git config --global user.email "your@email.com"  Should match with email associated with your GitHub account.)

**Reply by AleksBulajic** [reply]
Thank you so much for you help!!
it started counting again my commits and showing them on the contribution board!!
💪🏼
